CHANGELOG.md

```diff
@@ -50,7 +51,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Rename `shadow` to `shadow-sm`, `shadow-sm` to `shadow-xs`, and `shadow-xs` to `shadow-2xs` ([#14849](https://github.com/tailwindlabs/tailwindcss/pull/14849))
 - Rename `inset-shadow` to `inset-shadow-sm`, `inset-shadow-sm` to `inset-shadow-xs`, and `inset-shadow-xs` to `inset-shadow-2xs` ([#14849](https://github.com/tailwindlabs/tailwindcss/pull/14849))
 - Rename `drop-shadow` to `drop-shadow-sm` and `drop-shadow-sm` to `drop-shadow-xs` ([#14849](https://github.com/tailwindlabs/tailwindcss/pull/14849))
-- Rename `rounded` to `rounded-sm` and `rounded-sm` to `rounded-xs` ([#14849](https://github.com/tailwindlabs/tailwindcss/pull/14849))
```

Comment:
With the change of moving the old scale to be deprecated as it was, I think we can revert this one, right?

Reply:
This mostly affects the codemod stuff, I think instead of mapping `rounded` to `rounded-sm` etc. we should map it to the new `radius-*` utilities but leave the values as-is 🤔

Reply:
Ah wait this is more complicated, we can only migrate to `radius-*` when we the value is defined in the JS config, the compat layer only works `rounded-*` utilities

Reply:
I think the v4 equivalent should be:

```
rounded => radius-1
rounded-sm => radius-0.5
rounded-md => radius-1.5
etc.
```